### PR TITLE
chore: update react and happo packages

### PR DESCRIPTION
**Pull request by separatio**
[FX-2778]

### Description

As per Dependabot's open PRs, I've updated all selected packages for this week, except https://github.com/toptal/picasso/pull/2769.

### How to test

- all tests pass
- Click around Storybook

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Insert screenshots or screen recordings | Insert screenshots or screen recordings |

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure that tests pass by running `yarn test`
- [x] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-2778]: https://toptal-core.atlassian.net/browse/FX-2778?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .changeset/chatty-timers-pay.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+'@toptal/picasso': patch
+---
+
+---
```
Comment (ozgurkececioglu) [inline .changeset/chatty-timers-pay.md:5]: If you don't have any specific component to mention, you don't need this separator.
```suggestion
```

**Review by TomasSlama — COMMENTED** [review]
I think we don't need any changeset for this one. We bump only `devDependencies` so it does not affect our users.

File: .changeset/chatty-timers-pay.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Updated packages in Picasso as follows below
```
Comment (LashaJini) [inline .changeset/chatty-timers-pay.md:7]: Past simple.

**Review by OndrejTuma — APPROVED** [review]
Storybook looks OK 👍  I agree with Tomas that for this change we probably dont need changeset

**Review by LashaJini — APPROVED** [review]
Tested locally with no issues :+1:

**Comment by separatio** [comment]
Good point. Didn't think about devDependencies. I'll remove the changeset.

**Review by ozgurkececioglu — APPROVED** [review]

**Review by denieler — APPROVED** [review]

**Review by TomasSlama — APPROVED** [review]

**Pull request merged**